data-products/src/main/scala/org/sunbird/analytics/job/report/StateAdminReportJob.scala

```diff
@@ -194,34 +194,45 @@ object StateAdminReportJob extends optional.Application with IJob with StateAdmi
         
         saveUserSummaryReport(shadowDataSummary, channelSlugDF, storageConfig)
         saveUserDetailsReport(shadowUserStatusDF, channelSlugDF, storageConfig)
-
+    
+        generateTenantUserReport(organisationDF.toDF());
+    }
+    
+    def generateTenantUserReport(organisationDF: DataFrame) (implicit sparkSession: SparkSession, fc: FrameworkContext) = {
+    
+        val custodianOrgId = getCustodianOrgId();
+        val claimedUserDF = loadData(sparkSession, Map("table" -> "user", "keyspace" -> sunbirdKeyspace), None).where(col("rootorgid").isNotNull and !col("rootorgid").contains(custodianOrgId))
```

Comment:
pls name the var as usersDF - not just claimed.
This is just a report of users associated with state. Lets not use the word 'claim'.